streaming/base/format/mds/encodings.py

```diff
@@ -385,6 +386,56 @@ def __init__(self):
         super().__init__(np.float64)
 
 
+class StrEncoding(Encoding):
+    """Base class for stringified types.
+
+    Using variable-length strings allows us to store scalars with arbitrary precision.
+
+    The encode/decode methods of subclasses are the same except for typing specializations.
+    """
+
+    pass
+
+
+class StrInt(StrEncoding):
+    """Store int as variable-length digits str."""
+
+    def encode(self, obj: int) -> bytes:
+        self._validate(obj, int)
```

Comment:
Missing method comments. Check all the other places too!

Reply:
@karan6181  Looks like for simple methods with less than 3 lines in this file, encode and decode tend to not have doc strings. Shall I add doc strings to all of them?